indexer/packages/postgres/src/stores/fill-table.ts

```diff
@@ -49,10 +50,11 @@ export async function findAll(
     createdOnOrAfter,
     clientMetadata,
     fee,
+    page,
```

Comment:
Ideally, we'd have a check that if page is set, then orderBy must contain eventId. Can you add a TODO for now?

Reply:
Done!